repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/string/Base64Helper.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+
+package org.openrepose.commons.utils.string
+
+import java.util.Base64
+import java.nio.charset.StandardCharsets.UTF_8
+
+object Base64Helper {
+  def base64EncodeUtf8(encodeString: String): String = {
+    Base64.getEncoder.encodeToString(encodeString.getBytes(UTF_8))
```

Comment:
To summarize that, the only thing I would change here is not using the `encodeToString` method and instead doing it the hard way:
```
new String(Base64.getEncoder.encode(encodeString.getBytes(UTF_8)), StandardCharsets.ISO_8859_1)
```

Although, that is _precisely_ what the deprecated `encodeToString` does, so really, we don't _have_ to do anything here. Still, using deprecated methods when we don't need to isn't my favorite.

Reply:
The `encodeToString()` method isn't itself deprecated, they just have the `@SuppressWarnings("deprecation")` turned on since they haven't fixed the underlying implementation yet.